### PR TITLE
Alt parallel perf

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
r? @ghost

### CODE REVIEW

**Comment by Mark-Simulacrum** [comment]
@bors try @rust-timer queue

This is a parallel build of the compiler with -Zthreads=1 as per the default. So, this is testing sequential overhead.

File: src/ci/docker/dist-x86_64-linux/Dockerfile [hunk 1]
```diff
@@ -103,6 +103,7 @@ ENV RUST_CONFIGURE_ARGS \
       --set target.x86_64-unknown-linux-gnu.ranlib=/rustroot/bin/llvm-ranlib \
       --set llvm.thin-lto=true \
       --set rust.jemalloc
```
Comment (mati865) [inline src/ci/docker/dist-x86_64-linux/Dockerfile]:
```suggestion
      --set rust.jemalloc \
```

**Comment by Mark-Simulacrum** [comment]
@rust-timer build dc78b8ba143915e07375e9d7f05838222cb1db3e

**Comment by bjorn3** [comment]
Is -Zself-profile not working with parallel rustc? p.r-l.o says there are no self profile results.

**Comment by Mark-Simulacrum** [comment]
> Is -Zself-profile not working with parallel rustc? perf.r-l.o says there are no self profile results.

self-profile support was temporarily broken in perf.rlo's collector backend; I plan to rerun this benchmark in a day or so (I want the base master commit to have the self profile data).

**Comment by spastorino** [comment]
Benchs doesn't look too bad to be the first one that's run :)

**Comment by Mark-Simulacrum** [comment]
Self-profile appears to be currently broken on master, so let's just punt on getting those results, I may circle back to them at a later point.

@bors try @rust-timer queue

This is for a -Zthreads=8 build, which is the number of vCPUs on the perf server, which has a 4C/8T configuration.

**Comment by Mark-Simulacrum** [comment]
@rust-timer build f0b7b0a9327d3b43aa45a89e90d9785a06059b5a

**Comment by bjorn3** [comment]
There are a lot of


```
thread \'main\' panicked at \'assertion failed: `(left == right)`
  left: `\"codegen_and_optimize_crate\"`,
 right: `\"codegen_crate\"`\', summarize/src/analysis.rs:121:17
```

for the base commit.

**Comment by Mark-Simulacrum** [comment]
gah, I'll make sure we recollect it then. I was hopeful we were past that :)

**Comment by Mark-Simulacrum** [comment]
oh, we're already re-collecting that

**Comment by Mark-Simulacrum** [comment]
@bors try @rust-timer queue

Collecting data for 4 cores...

**Comment by Mark-Simulacrum** [comment]
Closing as we've finished gathering performance stats here for now...

**Pull request closed without merging**